lib/pull-manager.js

```diff
@@ -77,3 +78,13 @@ pullQueue.on('pullsChanged', function(pulls) {
       });
    });
 });
+
+// Cull old closed pulls from memory every once in a while
+setTimeout(() => {
+   const oldestAllowed = pullManager.getOldestAllowedPullTimestamp() * 1000;
+   pulls = pulls.filter((pull) =>
+      pull.isOpen() ||
+      !pull.data.closed_at ||
+      pull.data.closed_at.getTime() > oldestAllowed
```

Comment:
I'm confused to how `closed_at` has the function `getTime()` since it looks like `pull.data` is of type `PullData` which I then see has `DateString` as the type for `closed_at`?

![closed_at](https://user-images.githubusercontent.com/22064420/165973432-3b250205-fbb7-4d45-a070-38f5912d5640.png)

Reply:
Sorry for the confusion. All code outside of `frontend/` is running on the backend. Thus it has different models / types / no typescript. This is a `models/pulls.js` object and thus `pull.data.closed_at` is a `DateTime`.